test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -148,6 +150,17 @@ minerKeysetTest = do
 
     badMiner = Miner (MinerId "miner") $ MinerKeys $ mkKeySet ["bad-bad-bad"] "keys-all"
 
+txTimeoutTest :: PactTestM ()
+txTimeoutTest = do
+  -- get access to `enumerate`
+  runToHeight 20
+  handle (\(TxTimeout _) -> return ()) $ do
+    runBlockTest 
+      -- deliberately time out in newblock
+      [PactTxTest (buildBasicGas 1000 $ mkExec' "(enumerate 0 999999999999)") (\_ -> error "tx succeeded")] 
+    -- assert that the block is never produced
+    error "block succeeded"
```

Comment:
should be `assertFailure` (in general if you see you're writing a comment with "assert" in it ... :) )